src/ClientRuntime/Microsoft.Rest.ClientRuntime.Azure/CloudErrorJsonConverter.cs

```diff
@@ -50,6 +50,11 @@ public override bool CanConvert(Type objectType)
                 throw new ArgumentNullException("serializer");
```

Comment:
Per Hovsep's comment, please update the version in project.json and assemblyinfo.cs so that a new version can be published.